Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/BaseTeleOp.java

```diff
@@ -40,7 +44,12 @@ public void driveMechanism() {
         if (gamepad2.dpad_up) {
             motorSlideLeft.setPower(mechanismSpeed);
             motorSlideRight.setPower(mechanismSpeed);
-        } else if (gamepad2.dpad_down) {
+        } else if(gamepad2.dpad_down
+            && motorSlideLeft.getCurrentPosition() > bottomMotorSlideLeft
+                && motorSlideRight.getCurrentPosition() > bottomMotorSlideRight) {
+
+        }
+        else if (gamepad2.dpad_down) {
```

Comment:
This is not really the best way to have your software stop. It would be better to only move the motors down if the dpad down is pressed and the motor positions are less than the bottom. It needs to be less than because going up is a negative encoder value. Having an else if that does nothing is not good practice and very confusing.

Reply:
Since, the software stop works now, should we still change it?

Reply:
You already changed it to be correct.